VirtualSatelliteCAD/test/json_io/products/test_json_product_assembly_tree_traverser.py

```diff
@@ -81,11 +82,11 @@ def test_traverse_and_parse_json_tree(self):
 
         # this should have similar results to test_create_part_product_assembly_and_subassembly_with_root_part_manual in TestJsonProductAssembly
         active_document = ActiveDocument(self._WORKING_DIRECTORY).open_set_and_get_document(
-            PRODUCT_IDENTIFIER + "BasePlateBottom2_e8794f3d_86ec_44c5_9618_8b7170c45484")
+            PRODUCT_IDENTIFIER + BASEPLATEBOTTOM2_UNIQ_NAME)

```

Comment:
You are resusing PRODUCT_IDENTIFIER + BASEPLATEBOTTOM2_UNIQ_NAME, 
PRODUCT_IDENTIFIER + BASEPLATEBOTTOM1_UNIQ_NAME just create a variable for it